src/app/zap-templates/zcl/zcl.json

```diff
@@ -150,6 +150,66 @@
             "CurrentHeapHighWatermark",
             "ThreadMetrics"
         ],
+        "Thread Network Diagnostics": [
+            "channel",
```

Comment:
We can't do this, because `ThreadDiagosticsAttrAccess::Read` does fallback to the attribute store if `CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE` or `CHIP_ERROR_NOT_IMPLEMENTED` is returned....  We need to change it to not do that if we're going to add things here.

Reply:
acked